core/email_api.php

```diff
@@ -1549,11 +1549,13 @@ function email_format_bug_message( array $p_visible_bug_data ) {
 			$t_time_tracking = '';
 		}
 
+		$t_access_level_string = ' - ';
```

Comment:
Looks like the ' - ' should be part of the string concatenation below and not the access level string.  I know it wasn't part of your change.  But before we used to skip this ' - ' in some cases, now you seem to always have it.